src/libraries/System.Data.Odbc/src/System/Data/Odbc/OdbcConnectionFactory.cs

```diff
@@ -56,18 +56,15 @@ internal override DbConnectionPoolGroupProviderInfo CreateConnectionPoolGroupPro
         protected override DbMetaDataFactory CreateMetaDataFactory(DbConnectionInternal internalConnection, out bool cacheMetaDataFactory)
         {
             Debug.Assert(internalConnection != null, "internalConnection may not be null.");
+#pragma warning disable IDE0059
             cacheMetaDataFactory = false;
+#pragma warning restore IDE0059
```

Comment:
We can just remove this one.  It's internal and we can see the sole call site doesn't care about the value in the case of an exception.  We should be able to remove the out parameter entirely, I think.

Reply:
It's used by https://github.com/dotnet/runtime/blob/57bfe474518ab5b7cfe6bf7424a79ce3af9d6657/src/libraries/Common/src/System/Data/ProviderBase/DbConnectionFactory.cs#L391 for some odd singleton protection

Reply:
Yes, but that's always going to be true: either CreateMetaDataFactory throws or allowCache is true upon CreateMetaDataFactory exit.

Reply:
Makes sense